WordPress/gutenberg
# Allow custom CSS for theme.json elements

## What?

Enables CSS in theme.json styles > elements > CSS
Closes https://github.com/WordPress/gutenberg/issues/48439

## Why?

Custom CSS under theme.json styles.elements is not working, even though the schema says it works.

## How?

Updates `class WP_Theme_JSON_Gutenberg ` with a condition that checks if there are any items under styles > elements, locates custom css, and adds it to the custom CSS stylesheet.
Updates the global styles output to loop through the elements, locate the CSS, and add the css to the styles array.

## Testing Instructions
Activate Twenty Twenty-Three or Twenty Twenty-Four.
In theme.json, under styles > elements > heading, add the following as the first object:

`
"css": "border:3px solid orange;",
`

Example from TT4:

```
			"heading": {
				"css": "border:3px solid orange;",
				"color": {
					"text": "var(--wp--preset--color--contrast)"
				},
				"typography": {
					"fontFamily": "var(--wp--preset--font-family--heading)",
					"fontWeight": "400",
					"lineHeight": "1.2"
				}
			},
```

Next, open the Site editor, block editor, and the front of the website and confirm that headings have an orange border.


## Screenshots or screencast
